### DISCUSSION TITLE
Problem Accessing Prometheus Endpoint on Promitor Scraper via Docker

### DISCUSSION THREAD

**Discussion by yazidmissaoui** [tomkerkhove/promitor#2357] [Q&A] [answered]
Hello,
I am trying to deploy Promitor Scraper via Docker on my local machine, but I am having trouble accessing the Prometheus endpoint (http://localhost:8999/metrics). I get an HTTP ERROR 404 in my browser when I try to access this endpoint.

Here are the relevant pieces of my setup:

I am running the Docker container using the following command:

`docker run -it --rm -p 8999:80 --name promitor-agent-scraper --env-file E:\promitor\az-mon-auth.creds --volume E:\promitor\metrics-declaration.yaml:/config/metrics-declaration.yaml --volume E:\promitor\runtime.yaml:/config/runtime.yaml ghcr.io/tomkerkhove/promitor-agent-scraper:2.8.0`

**My runtime.yaml configuration is as follows:**
![image](https://github.com/tomkerkhove/promitor/assets/38033421/84ee6936-5164-4328-a5b1-97ff515ec764)

**My metrics-declaration.yaml configuration is as follows:**
![image](https://github.com/tomkerkhove/promitor/assets/38033421/7b87df89-2ea8-4809-a357-13101f16e24c)

The application log shows that Promitor is starting up and running normally, with no errors reported.²
![image](https://github.com/tomkerkhove/promitor/assets/38033421/152e719c-d518-42b2-8a39-c1c044143c2d)

I have verified that Docker port mapping is working correctly by running and accessing an nginx image via Docker.

I am looking for guidance on how to resolve this issue. Any help would be greatly appreciated.

**Comment by tomkerkhove** [comment, marked as answer]
Sorry for the delay but I'm on holiday - It looks like you're using `metricSink` instead of `metricSinks`. Can you update that and try again please?

**Reply by yazidmissaoui** [reply]
Thank for your answer. Now it's OK.

**Reply by tomkerkhove** [reply]
Good to hear!
